src/ripple/protocol/impl/Issue.cpp

```diff
@@ -43,60 +43,33 @@ operator<<(std::ostream& os, Issue const& x)
     return os;
 }
 
-/** Ordered comparison.
-    The assets are ordered first by currency and then by account,
-    if the currency is not XRP.
-*/
-int
-compare(Issue const& lhs, Issue const& rhs)
+std::weak_ordering
+operator<=>(Issue const& lhs, Issue const& rhs)
 {
-    int diff = compare(lhs.currency, rhs.currency);
-    if (diff != 0)
-        return diff;
+    auto const cmpCcy{lhs.currency <=> rhs.currency};
+    if (cmpCcy != std::weak_ordering::equivalent)
+    {
+        return cmpCcy;
+    }
     if (isXRP(lhs.currency))
-        return 0;
-    return compare(lhs.account, rhs.account);
+    {
+        return std::weak_ordering::equivalent;
+    }
+    else
+    {
+        return (lhs.account <=> rhs.account);
+    }
```

Comment:
Same as above: 

- Move to the header and mark `inline`.
- Remove the curlies and avoid the `else`.

Reply:
Fixed.